integrations/huggingface-transformers/README.md

```diff
@@ -62,129 +96,207 @@ output:
-    --dataset_name squad \
-    --per_device_train_batch_size 12 \
-    --per_device_eval_batch_size 24 \
-    --preprocessing_num_workers 6 \
-    --do_train \
-    --do_eval \
-    --evaluation_strategy epoch \
-    --fp16 \
-    --seed 21636  \
-    --per_device_train_batch_size 16 \
-    --per_device_eval_batch_size 24 \
-    --preprocessing_num_workers 6 \
-    --save_strategy epoch \
-    --save_total_limit 1
+sparseml.transformers.text_classification \
+  --output_dir pruned_quantized_obert-text_classification_sst2 \
+  --model_name_or_path zoo:nlp/masked_language_modeling/obert-base/pytorch/huggingface/wikipedia_bookcorpus/pruned90-none \
+  --recipe zoo:nlp/sentiment_analysis/obert-base/pytorch/huggingface/sst2/pruned90_quant-none \
+  --distill_teacher zoo:nlp/sentiment_analysis/obert-base/pytorch/huggingface/sst2/base-none \
+  --task_name sst2 --max_seq_length 128 --per_device_train_batch_size 32 --per_device_eval_batch_size 32 --preprocessing_num_workers 6 \
+  --do_train --do_eval --evaluation_strategy epoch --fp16  \
+  --save_strategy epoch --save_total_limit 1
 ```
 
-### Exporting to ONNX
+The script uses the SparseZoo stubs to identify and download the starting checkpoint and YAML-based recipe file from the SparseZoo. SparseML parses the transfer learning recipe and adjusts the trainign process to maintain sparsity during the fine-tuning process.
+
+The resulting model is 90% pruned and quantized, and achieves 92% validation accuracy on SST2!
+
+Keep in mind that the `--distill_teacher` argument is set to pull a dense SST2 model from the SparseZoo. If you trained a dense teacher with the command from above, update the script to use `--distill_teacher ./dense_obert-text_classification_sst2`.
 
-The DeepSparse Engine uses the ONNX format to load neural networks and then deliver breakthrough performance for CPUs by leveraging the sparsity and quantization within a network.
+### Export to ONNX
 
-The SparseML installation provides a `sparseml.transformers.export_onnx` command that you can use to load the training model folder and create a new model.onnx file within. Be sure the `--model_path` argument points to your trained model. By default, it is set to the result from transfer learning a sparse-quantized BERT model:
+The SparseML installation provides a `sparseml.transformers.export_onnx` command that you can use to export the model to ONNX. Be sure the `--model_path` argument points to your trained model:
 
 ```bash
 sparseml.transformers.export_onnx \
-    --model_path models/sparse_quantized \
-    --task 'question-answering' \
-    --sequence_length 384
+    --model_path ./pruned_quantized_obert-text_classification_sst2 \
+    --task text_classification
 ```
 
-### DeepSparse Engine Deployment
+The command creates a `./deployment` folder in your local directory, which contains the ONNX file and necessary Hugging Face tokenizer and configuration files.
```

Comment:
The command creates a `./deployment` folder in your local directory, containing the ONNX file and necessary Hugging Face tokenizer and configuration files.